src/python/pants_release/generate_release_announcement.py

```diff
@@ -1,16 +1,20 @@
 # Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
+
 import argparse
 import json
+import os
 
 from pants_release.common import VERSION_PATH, sorted_contributors
 from pants_release.git import git
 
+from pants.util.dirutil import safe_mkdir
 from pants.util.strutil import softwrap
 
+version = VERSION_PATH.read_text().strip()
```

Comment:
SCREAM_CASE?

Reply:
I don't understand the question

Reply:
If it's that version should be VERSION - I don't think it matters.

Reply:
Yeah. That's the convention in Python. And conventions are great, because they help relaxing the parsing required for the human to read the code.

Personally, I know instantly when something is screaming case it's a global. Otherwise, it'd take me a bit to find where the variable is coking from.

Reply:
Changed